x-pack/test/functional_with_es_ssl/apps/cases/edit_case_form.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import uuid from 'uuid';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  const common = getPageObject('common');
+  const header = getPageObject('header');
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const cases = getService('cases');
+  const retry = getService('retry');
+  const comboBox = getService('comboBox');
+
+  describe('Edit case', () => {
+    // create the case to test on
+    before(async () => {
+      await common.navigateToApp('cases');
+      await cases.api.createNthRandomCases(1);
+    });
+
+    after(async () => {
+      await cases.api.deleteAllCases();
+    });
+
+    beforeEach(async () => {
+      await common.navigateToApp('cases');
+      await cases.common.goToFirstListedCase();
+      await header.waitUntilLoadingHasFinished();
+    });
+
+    it('edits a case title from the case view page', async () => {
+      const newTitle = `test-${uuid.v4()}`;
+
+      await testSubjects.click('editable-title-edit-icon');
+      await testSubjects.setValue('editable-title-input-field', newTitle);
+      await testSubjects.click('editable-title-submit-btn');
+
+      // wait for backend response
+      await retry.tryForTime(5000, async () => {
+        const title = await find.byCssSelector('[data-test-subj="header-page-title"]');
+        expect(await title.getVisibleText()).equal(newTitle);
+      });
+    });
+
+    it('adds a comment to a case', async () => {
+      const commentArea = await find.byCssSelector(
+        '[data-test-subj="add-comment"] textarea.euiMarkdownEditorTextArea'
+      );
+      await commentArea.focus();
+      await commentArea.type('Test comment from automation');
+      await testSubjects.click('submit-comment');
+
+      // validate user action
+      await find.byCssSelector(
```

Comment:
Does this validate that the correct comment has been added?

Reply:
I added the validation